components/basic-filters/src/main/java/org/datacleaner/beans/filter/DateRangeFilter.java

```diff
@@ -71,4 +75,50 @@ public Date getLowestValue() {
     public int compare(Date o1, Date o2) {
         return o1.compareTo(o2);
     }
+
+    @Override
+    public String getSuggestedLabel() {
+        final Date highestValue = getHighestValue();
+        final Date lowestValue = getLowestValue();
+        
+        if (highestValue == null || lowestValue == null) {
+            return null;
+        }
+        
+        final InputColumn column = getColumn();
```

Comment:
InputColumn is a raw type. References to generic type InputColumn<E> should be parameterized.